api/config/rolepermission.php

```diff
@@ -228,6 +228,10 @@
             'en' => 'View any submitted Applications',
             'fr' => 'Voir n\import quelle candidature soumises à n\'importe quel bassin.'
         ],
+        'view-own-applicantProfile' => [
+            'en' => 'View own Applicant Profile.',
+            'fr' => 'Voir son propre profil de candidat.'
+        ],
```

Comment:
should probably also update RolePermissionTest.php with this
ensuring we have 100% (ideally) coverage of every added role